src/_includes/layouts/base.njk

```diff
@@ -290,14 +290,15 @@ document.getElementById('nav-toggle').onclick = function(){
 <script async src="https://cdn.jsdelivr.net/npm/lite-youtube-embed@0.2.0/src/lite-yt-embed.min.js"></script>
 <link rel="preload" href="https://cdn.jsdelivr.net/npm/lite-youtube-embed@0.2.0/src/lite-yt-embed.min.css" as="style" onload="this.onload=null;this.rel='stylesheet'" crossorigin="anonymous" referrerpolicy="no-referrer" />
 
+<!-- Cookie banner -->
+<script async type="text/javascript" src="/js/cc.min.js"></script>
+
+{%- if DEV_MODE -%}
```

Comment:
and then 
```suggestion
{%- if not DEV_MODE -%}
```

Reply:
It just feels weird to read that the tracking will only happen when in dev_mode, but dev_mode means not dev mode